static/app/components/events/rrwebIntegration.tsx

```diff
@@ -23,7 +23,7 @@ class RRWebIntegration extends AsyncComponent<Props, State> {
       [
         'attachmentList',
         `/projects/${orgId}/${projectId}/events/${event.id}/attachments/`,
-        {query: {query: 'rrweb.json'}},
+        {query: {query: 'rrweb'}},
```

Comment:
this will return all attachments that includes `rrweb` anywhere in the string -- we may want to tighten this up a bit in the future?

Reply:
I'd just put your GitHub comment in code, `Note: This will return all attachments ...`. Seems fair to put that warning there?